src/components/pages/NavBar.tsx

```diff
@@ -0,0 +1,29 @@
+import classNames from "classnames";
+import { NavLink } from "react-router-dom";
+
+export const Navbar = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean}) => classNames('navbar-item',{
+    'has-background-grey-lighter': isActive,
+  });
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink to="/" className={({ isActive }) => getLinkClass({ isActive })}>
+            Home
+          </NavLink>
```

Comment:
You are using the `NavLink` component in a way that is likely not working as you expect. The `className` prop expects a string, but you are passing a function that returns a string. The function you pass to `className` is not being called with an `isActive` argument, so it will always be `undefined`. If you want to conditionally style your `NavLink` based on whether it is active, you should use the `activeClassName` prop instead.